src/main/src/main/java/org/geoserver/system/status/MetricInfo.java

```diff
@@ -18,7 +18,7 @@
     LOGICAL_CPUS("CPU", 101, "Number of logical CPUs"),
     RUNNING_PROCESS("CPU", 102, "Number of running process"),
     RUNNING_THREADS("CPU", 103, "Number of running threads"),
-    CPU_LOAD("CPU", 104, "CPU average load", "%"),
+    // 104 removed from OSHI
```

Comment:
What was removed was OSHI tracking its own ticks here. You could keep track of the ticks yourself imilar to how you do for `PER_CPU_LOAD` and keep this.